### DISCUSSION TITLE
2021-08-25 Weekly Meeting 18 Agenda

### DISCUSSION THREAD

**Discussion by humbletim** [omigroup/omigroup#79] [Weekly Meetings]
Weekly Meeting 18 is happening on Wednesday 2021-08-25 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
   - Welcome new members and attendance
   - [OMI Survey](https://forms.gle/Dxvap4auBGim6JsL9)
   - Working agreements / playbook follow-up
   - Discord Role Bot (#roles in Discord)
   - Website
       - update on iterative development and release strategy
       - call out for website section maintainers and community content
   - glTF Sub Group
       - progress updates
       - call out for extension brainstorming [list](https://github.com/omigroup/gltf-extensions/discussions/7)
       - Khronos 3D Formats Group Liaison
   - Virtual World URIs / omi-world-metadata
        - follow-ups from [2021.08.21 hack-together](https://discord.com/channels/770382203782692945/878654908549713970)
   - World Exploration
       - discoveries from [Tuesday's vket outing](https://discord.com/channels/770382203782692945/852608195972694067)
   - Demo Night coordination follow-up
     - https://forms.gle/CknpQQKNi7ywQu9u5

Add ons: 
  - ...

**Comment by mrmetaverse** [comment]
@humbletim and I were talking and eventually, we may want to look out our Main (formerly known as "Master") backlog at the end of our meetings. This is where we can track what working groups we have, which projects are active, and which are for the future. 

If a topic gets removed from the agenda, it is often forgotten by the next week, because we usually copy and paste the agenda from the previous week. This may cause us to lose track of projects. We could explicitly state "We haven't seen any progress on the OMI place to meet, work, and hang, in about 4 weeks. So I am de-prioritizing #44 from active, to the backlog." And when a new topic gets added, we would explicitly state "We recently accepted proposal #49 and are moving this to active". 

That way when new topics are introduced we can ensure they are tracked/trackable. And when old topics fall to the wayside, we don't forget about them later. 

I am suggesting 5-10 min at the beginning or end of our Wednesday all-hands meetings. 

Thoughts?

**Reply by stonedrew** [reply]
Keeping a cadence of this is good practice! Additionally, doing a quick (limited by timebox ox or # of issues?) backlog refinement at our largest meetings can be a show-and-tell to let others see how the sausage is made, and demystify the process.
I recommend at the end of the meeting, added as an agenda item.

**Comment by mrmetaverse** [comment]
Next week, let's start or end our meeting with a quick 15-20 min retro with those who can make it. 
- What has been going well? 
- What could be improved? 
- What could we do different moving forward? 

We will take 5 min to answer in a figjam board. And review for a timeboxed 10-15 minutes. We feed on actionable feedback.
